lib/ecto/adapters/postgres.ex

```diff
@@ -25,6 +25,22 @@ defmodule Ecto.Adapters.Postgres do
 
       YourApp.Repo.all(Queryable, prepare: :unnamed)
 
+  ### Migration options
+
+    * `:migration_lock` - prevent multiple nodes from running migrations at the same
+      time by obtaining a lock. The value `true` will lock migrations by wrapping
+      the entire migration inside a database transaction, including inserting the
+      migration version into the migration source (by default, "schema_migrations").
+      You may alternatively select `:pg_advisory_lock` which has the benefit
+      of allowing concurrent operations such as creating indexes. (default: true)
```

Comment:
Can we have the default be `:table_lock` instead of true? :)

Reply:
ah I thought `true` was being set somewhere, but it's not... it's actually `nil` so yeah `:table_lock` is much better.